### DISCUSSION TITLE
部分文献的超链接失效。

### DISCUSSION THREAD

**Discussion by earthmoon** [hushidong/biblatex-gb7714-2015#174] [Q&A]
您好，

用的武大模板，采用`biblatex`，`xelatex`爆内存就用了`lualatex-biber-lualatex-lualatex`。编译后，发现部分中文文献点击时，不能跳转到文末参考文献。一开始我以为是全部中文是这样，实际和`Lualatex`有关。使用`xelatex`不会有这一问题。我无法在武大模板中复现这个问题，因为我在武大模板的基础上添加了很多宏包之类的。有没有办法通过log文件查找哪里出了问题？

谢谢！

**Comment by earthmoon** [comment]
我还是用`xelatex`，搞不懂`lualatex`，占内存，又慢，怎么还会有这个怪异问题。

**Comment by hushidong** [comment]
这个可能与hyperref相关，需要深入查才行。

你最好还是把它简化成一个最小例子来测试一下。只跟biblatex相关，用lualatex编译，测试。

我由于基本不用lualatex所以没有遇到过这种情况。我一般中文用xelatex，西文用pdflatex，lualatex只在xelatex实在不能用时测试一下看看问题。

**Reply by earthmoon** [reply]
试了很久，做不出例子，我的文档运行估计要20分钟，比较大。目前`xelatex`可以规避这个问题。要是有时间的话，可否看下这个[问题](https://github.com/whutug/whu-thesis/discussions/230)。文内引用时，两个作者姓之间出现逗号。我也不确定按照期刊排版，怎么表示才更为学术一点。
